index.html

```diff
@@ -31,6 +31,142 @@ <h2>I’m Ismail Glad to see you!</h2>
+                    <span>2015</span>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
+                </ul>
+                <button>See Project</button>
+            </section>
+            <section class="work">
+                <img src="images/Snapshoot-Portfolio-3.png" alt="">
+                <h3>Tonic</h3>
+                <div class="counter">
+                    <span>CANOPY</span>
+                    <span class="counters"></span>
+                    <span>Back End Dev</span>
+                    <span class="counters"></span>
+                    <span>2015</span>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
+                </ul>
+                <button>See Project</button>
+            </section>
+            <section class="work">
+                <img src="images/Snapshoot-Portfolio-4.png" alt="">
+                <h3>Multi-Post Stories</h3>
+                <div class="counter">
+                    <span>CANOPY</span>
+                    <span class="counters"></span>
+                    <span>Back End Dev</span>
+                    <span class="counters"></span>
+                    <span>2015</span>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
+                </ul>
+                <button>See Project</button>
+            </section>
+            <section class="about-myself">
+                <p class="myself">About Myself</p>
+                <p class="content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="connect">LET'S CONNECT</p>
+                <div class="social-media">
+                    <ul>
+                        <li><i class="fa fa-github"></i></li>
+                        <li><i class="fa fa-linkedin"></i></li>
+                        <li><i class="fa fa-angellist"></i></li>
+                        <li><i class="fa fa-twitter"></i></li>
+                        <li><i class="fa fa-medium"></i></li>
+                    </ul>
+                </div>
+                <button>Get my resume</button>
```

Comment:
Your About section looks a bit different from the given design kindly take note of the spacings, font properties and make the necessary changes👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="300" alt="yours" src="https://user-images.githubusercontent.com/64643995/234843342-de40d01b-de99-4530-a32a-a28311e5db0e.png">|<img width="300" alt="design" src="https://user-images.githubusercontent.com/64643995/234843406-0d08fc35-78d2-4d60-a40e-04474983c9c6.png">|

Reply:
@Ibizugbe Ibizugbe, my gratitude knows no bounds for your thorough feedback, thank you once again